lib/migration-stream.js

```diff
@@ -28,21 +29,44 @@ function MigrationStream() {
         return layer.version === 2;
     });
 
+    var newValidatorErrors; 
```

Comment:
@millzpaugh A bit subjective (except for the error handling), so feel free to disregard. What about:
```js
if (v2 === true) {
  vtvalidate.isValid(tile.buffer, function(err, result) {
    if (err) return callback(err); // handle any unexpected errors
    
    var newValidatorErrors = result;
    var err = checkForErrors(info);
    var data = {};
    ...
  }
}
```

Reply:
updated!